modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsForm.es.js

```diff
@@ -756,28 +786,48 @@ class ResultRankingsForm extends Component {
 					valueMap={{
 						addedHiddenIds: this._getHiddenAdded(),
 						aliases,
+						disabled,
```

Comment:
No need `disabled`. This component is used for passing values in the submission of the form, so I don't think it would even know what `disabled` was for. 😅